packages/component/src/createLoadable.js

```diff
@@ -195,6 +195,13 @@ function createLoadable({ resolve = identity, render, onLoad }) {
       ctor.requireAsync(props)
     }
 
+    Loadable.load = props => {
+      if (typeof window === 'undefined') {
+        throw new Error('`load` cannot be called server-side')
```

Comment:
It is really has to be an error?

Reply:
Another issue, for now I will let it like this.

Reply:
I was happy to see this PR but looking at the code I can not use it because of this limitation.  I need as much client side as server.

Reply:
I removed it 🤗

Reply:
Thank you, that would be perfect with a method like isReady.

Reply:
Yep, but remember, if you need it, you are probably taking the wrong way to be Suspense ready. When Suspense will be released, you will not be able to wait for anything except in render.

Reply:
I know that Suspense does not allow it, but it's still useful information. In my case it allows me to display a progress bar when you have to load one or more chunks and not display it when they are already loaded.

Reply:
Ooff, how many chunks you have to load?